python/tests/erdos_to_ros_test.py

```diff
@@ -0,0 +1,173 @@
+"""Sends a erdos message to an operator that converts them
+to a ros message and publishes them.
+
+Uses a subscriber to test whether the converting operator
+is successful.
+"""
+
+from multiprocessing import Value
+import time
+import pytest
+
+import rospy
+from std_msgs.msg import String
+from rospy.exceptions import ROSException, ROSSerializationException
+
+import erdos
+
+
+ROSTOPIC = "test_erdos_to_ros"
+ROS_NODE_NAME = "erdos_to_ros_node"
+RECEIVED_MESSAGES = []
+NUM_RECEIVED = Value('i', 0)
+TEST_FAILED = Value('i', 0)
+
+
+class SendOp(erdos.Operator):
+    """Generates and sends a stream of Erdos messages.
+    Args:
+        write_stream: Stream to which messages are written
+        messages: List of Python primitives or objects that
+            are sent as Erdos messages in the write stream.
+    """
+
+    def __init__(self, write_stream, messages):
+        self.write_stream = write_stream
+        self.messages = messages
+        self.coord = 0
+
+    @staticmethod
+    def connect():
+        return [erdos.WriteStream()]
+
+    def run(self):
+        for raw_msg in self.messages:
+            msg = erdos.Message(erdos.Timestamp(coordinates=[self.coord]), raw_msg)
+            print("SendOp: sending {msg}".format(msg=msg))
+            self.write_stream.send(msg)
+            self.coord += 1
+            time.sleep(0.25)
+
+
+class ErdosToRosOp(erdos.Operator):
+    """Converts Erdos messages to ROS messages.
+    Args:
+        publish_stream (:py:class:`erdos.ReadStream`): Stream on which the
+            operator receives control commands.
+        ros_msg_type: Type used to publish to ros messages
+        rostopic: rostopic to publish messages to
+        node_name: Ros node name
+        func: Callback function that converts a single ros_msg to an erdos_msg
+    """
+
+    def __init__(self, publish_stream, ros_msg_type, rostopic, node_name, func):
```

Comment:
Can we name the function to `conversion_func` from `func`?

Also, we should probably move the order of these arguments around.